examples/jsm/loaders/KTX2Loader.js

```diff
@@ -259,13 +259,21 @@ class KTX2Loader extends Loader {
 	 * @param {object?} config
 	 * @return {Promise<CompressedTexture|DataTexture|Data3DTexture>}
 	 */
-	_createTexture( buffer, config = {} ) {
+	async _createTexture( buffer, config = {} ) {
```

Comment:
this line alone will do everything you need here — no need to explicitly catch and re-throw the error, after the function is async.

Reply:
Updated the PR.